job_templates/sag_pt_deploy_map/site-1/config_fed_client.conf

```diff
@@ -55,10 +55,10 @@
     }
   ],
 
-  # this defined an array of task data filters. If provided, it will control the data to server
+  # this defined an array of task data filters. If provided, it will control the data from server controller to client executor
   task_data_filters =  []
 
-  # this defined an array of task result filters. If provided, it will control the result to server
+    # this defined an array of task result filters. If provided, it will control the result from client executor to server controller
```

Comment:
No need to change.  But I just noticed that there is one extra space at the beginning of this line.